app/Command/Bundle.hs

```diff
@@ -3,141 +3,21 @@ module Command.Bundle (command) where
-    mid <- guessModuleIdentifier filename
-    length js `seq` return (mid, Just filename, js)                                            -- evaluate readFile till EOF before returning, not to exhaust file handles
-
-  let entryIds = map (`ModuleIdentifier` Regular) optionsEntryPoints
-
-  currentDir <- liftIO getCurrentDirectory
-  let outFile = if optionsSourceMaps then fmap (currentDir </>) optionsOutputFile else Nothing
-  let withRawModules = if optionsDebug then Just bundleDebug else Nothing
-  bundleSM input entryIds optionsMainModule optionsNamespace outFile withRawModules
-
--- | Print a JSON representation of a list of modules to stderr.
-bundleDebug :: (MonadIO m) => [Module] -> m ()
-bundleDebug = liftIO . hPutStrLn stderr . LBU8.toString . encodePretty' (defConfig { confCompare = keyComparer })
-  where
-  -- | Some key order hints for improved readability.
-  keyComparer :: Text -> Text -> Ordering
-  keyComparer =  keyOrder ["type", "name", "moduleId"]     -- keys to put first
-              <> flip (keyOrder ["dependsOn", "elements"]) -- keys to put last
-
--- | Command line options parser.
-options :: Parser Options
-options = Options <$> some inputFile
-                  <*> optional outputFile
-                  <*> many entryPoint
-                  <*> optional mainModule
-                  <*> namespace
-                  <*> sourceMaps
-                  <*> debug
-  where
-  inputFile :: Parser FilePath
-  inputFile = Opts.strArgument $
-       Opts.metavar "FILE"
-    <> Opts.help "The input .js file(s)"
-
-  outputFile :: Parser FilePath
-  outputFile = Opts.strOption $
-       Opts.short 'o'
-    <> Opts.long "output"
-    <> Opts.help "The output .js file"
-
-  entryPoint :: Parser String
-  entryPoint = Opts.strOption $
-       Opts.short 'm'
-    <> Opts.long "module"
-    <> Opts.help "Entry point module name(s). All code which is not a transitive dependency of an entry point module will be removed."
-
-  mainModule :: Parser String
-  mainModule = Opts.strOption $
-       Opts.long "main"
-    <> Opts.help "Generate code to run the main method in the specified module."
-
-  namespace :: Parser String
-  namespace = Opts.strOption $
-       Opts.short 'n'
-    <> Opts.long "namespace"
-    <> Opts.value "PS"
-    <> Opts.showDefault
-    <> Opts.help "Specify the namespace that PureScript modules will be exported to when running in the browser."
-
-  sourceMaps :: Parser Bool
-  sourceMaps = Opts.switch $
-       Opts.long "source-maps"
-    <> Opts.help "Whether to generate source maps for the bundle (requires --output)."
-
-  debug :: Parser Bool
-  debug = Opts.switch $
-       Opts.long "debug"
-    <> Opts.help "Whether to emit a JSON representation of all parsed modules to stderr."
+app :: IO ()
+app = do
+  hPutStrLn stderr $ unlines
+    [ "'purs bundle' was removed in the v0.15.0 release."
+    , "Use 'esbuild' or another bundler. For info on migrating to a bundler, see"
```

Comment:
Are we taking a stance on suggesting `esbuild`? I'm fine with this, but I'd like to make sure maintainers in general are OK with suggesting a specific bundler here.

Reply:
What about
```
[ "'purs bundle' was removed in the v0.15.0 release."
, "See https://www.github.com/purescript/documentation/migration-guides/0.15-Migration-Guide.md "
, "for more information and bundler alternatives."
]
```